test/cri-containerd/disable_vpmem_test.go

```diff
@@ -0,0 +1,107 @@
+//go:build functional
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/Microsoft/hcsshim/pkg/annotations"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// Use unique names for pods & containers so that if we run this test multiple times in parallel we don't
+// get failures due to same pod/container names.
+func uniqueRef() string {
+	t := time.Now()
+	var b [3]byte
+	// Ignore read failures, just decreases uniqueness
+	rand.Read(b[:])
+	return fmt.Sprintf("%d-%s", t.UnixNano(), base64.URLEncoding.EncodeToString(b[:]))
+}
+
+func Test_70LayerImagesWithNoVPmemForLayers(t *testing.T) {
+	requireFeatures(t, featureLCOW)
+
+	ubuntu70Image := "cplatpublic.azurecr.io/ubuntu70extra:18.04"
+	alpine70Image := "cplatpublic.azurecr.io/alpine70extra:latest"
+	testImages := []string{ubuntu70Image, alpine70Image}
+	pullRequiredLCOWImages(t, []string{imageLcowK8sPause, ubuntu70Image, alpine70Image})
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	nContainers := 4
+	podID := ""
+	containerIDs := make([]string, nContainers)
+
+	defer cleanupPod(t, client, ctx, &podID)
+	for i := 0; i < nContainers; i++ {
+		defer cleanupContainer(t, client, ctx, &containerIDs[i])
+	}
+
+	sandboxRequest := getRunPodSandboxRequest(
+		t,
+		lcowRuntimeHandler,
+		WithSandboxAnnotations(map[string]string{
+			annotations.VPMemCount:          "0",
+			annotations.PreferredRootFSType: "initrd",
+		}),
+	)
+	// override pod name
+	sandboxRequest.Config.Metadata.Name = fmt.Sprintf("pod-%s", uniqueRef())
```

Comment:
Can we include `t.Name()` in the pod/container names? That will make it easier to tell which test leaked them.